integration_tests/mockApis/auth.ts

```diff
@@ -118,20 +117,20 @@ const token = ({ authorities }: { authorities?: Array<ApplicationRoles> }) =>
         internalUser: true,
         scope: 'read',
         token_type: 'bearer',
-        user_name: mockedUsername(),
+        user_name: mockedUser.username,
       },
       status: 200,
     },
   })
 
 export default {
   getSignInUrl,
-  mockedUsername,
+  mockedUser,
```

Comment:
I thought the Cypress config would complain if this wasn't a function, but it seems to be okay